ManatiCR/retopais
# RP-56: Add missing prizes logos

https://manati.atlassian.net/browse/RP-56
## How to test
- `drush fra -y && drush cc all`
- Add some alliances of "Premios" type.
  Visit `/que-es-reto-pais`. You should see the logos of the alliances you added between the prizes info text and the Problem prizes block.
